pkg/apic/apiserver/models/management/v1alpha1/model_discovery_agent_status.go

```diff
@@ -31,4 +31,6 @@ type DiscoveryAgentStatus struct {
 	LastActivityTime time.Time `json:"lastActivityTime,omitempty"`
 	// Version name for the SDK revision.
 	SdkVersion string `json:"sdkVersion,omitempty"`
+	// Use for public key
+	PublicKey string `json:"publicKey,omitempty"`
```

Comment:
May be we need to regenerate the model ? This property was removed from api server definitions

Reply:
good call